kubernetes.yaml

```diff
@@ -58,11 +66,13 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: kubebox
+  namespace: kubebox
 ---
 kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: kubebox
+  namespace: kubebox
 subjects:
 - kind: ServiceAccount
   name: kubebox
```

Comment:
nit: could you please remove the `# TO BE UPDATED IF NEEDED` comment?

Reply:
I don't now what you mean, where did you see this comment?

Reply:
I mean this: https://github.com/astefanutti/kubebox/blob/a3a6e423aa84fb2b915b0fe82e395bdd61f0c38a/kubernetes.yaml#L69. Let me remove it after I merge your PR.